crates/cheatcodes/src/inspector.rs

```diff
@@ -83,14 +84,6 @@ pub struct BroadcastableTransaction {
 /// List of transactions that can be broadcasted.
 pub type BroadcastableTransactions = VecDeque<BroadcastableTransaction>;
 
-#[derive(Clone, Debug)]
-pub struct AccountAccess {
-    /// The account access.
-    pub access: crate::Vm::AccountAccess,
-    /// The call depth the account was accessed.
-    pub depth: u64,
-}
-
```

Comment:
Since the depth field is not exposed in the `Vm::AccountAccess` struct, we can remove this wrapper type.